deployment/k8s-config/kustomize/base/skaha/skaha-tomcat-deployment.yaml

```diff
@@ -40,7 +40,7 @@ spec:
           value: "ivo://cadc.nrc.ca/gms?skaha-users"
         - name: skaha.adminsgroup
           value: "ivo://cadc.nrc.ca/gms?skaha-admins"
-        image: images.canfar.net/skaha-system/skaha:0.10.0
+        image: images.canfar.net/skaha-system/skaha:0.10.1
```

Comment:
The current released version is 0.11.0, so you'll need to increment that one.  See the `VERSION` file.

Reply:
Yes, just finished doing another pull from master.

Reply:
Maybe go to 0.12.0 with this one as there are API additions

Reply:
Okay.